### ISSUE TITLE
exception by selecting model GPT-4

### ISSUE THREAD

**Issue by kannstDuMal** [ugorsahin/ChatGPT_Automation#27]
Super project!!! Thx.
In Chrome browser GPT-4 is correctly selected as default.

Ran correctly until now.
Since 1 day (?) unfortunately always the model GPT-3.5 is selected using this ChatGPT_Client.
The WebUI has changed a bit ;-(

calling 
`chatgpt.switch_model("GPT-4") `
returns the error message:
```
Exception has occurred: ElementClickInterceptedException       (note: full exception trace is shown but execution is paused at: _run_module_as_main)
Message: element click intercepted: Element <span class="truncate text-sm font-medium md:pr-1.5 pr-1.5">...</span> is not clickable at point (1177, 49). Other element would receive the click: <div class="grid-cols-[10px_1fr_10px] grid h-full w-full grid-rows-[minmax(10px,_1fr)_auto_minmax(10px,_1fr)] md:grid-rows-[minmax(20px,_1fr)_auto_minmax(20px,_1fr)] overflow-y-auto">...</div>
  (Session info: chrome=114.0.5735.133)
```
Would be great if this works again to use the pro account properly.

**Comment by ugorsahin** [comment]
Hello,

Can you try with the latest commit  [203a0a8](https://github.com/ugorsahin/ChatGPT_Automation/commit/203a0a8e0108cd91ea0fb7141e7e5c756555265a) and let me know if it is working on your end?

**Comment by kannstDuMal** [comment]
wow ... quick reaction ;-)
Unfortunately bad news.
Same problem, same exception. 
No change ;-(

**Comment by ugorsahin** [comment]
Try passing --no-cache-dir to pip install

**Comment by kannstDuMal** [comment]
the two files are up to date to your patch-level (i checked manually) and the debugger also the latest code is executed.

Unfortunately i am not so good at xpath.

Since mouse-over at "GPT-4" in Firefox as well as in Chrome a drop-downmenu appears ... could it be that 2 actions are necessary? 
1. mouse-over GPT-4
2. then select "Default"

**Comment by ugorsahin** [comment]
Hey, 

I don't have a premium account anymore so I am not sure if that functionality works. If you can provide screenshots of that and DOM, I can do my best to make sure that function is working. Otherwise, you are welcome to send a PR with a working version.

In my free account, that function opens a modal to funnel into the premium account subscription, that's why I thought it should have worked. However, it seems like there may be an extra step.

**Comment by kannstDuMal** [comment]
ok i got it.

The exception was raised by the "ChatGPT: Tips for getting started" Dialog.
The dialog must be closed first.


in login-function:
```
	# Find password textbox, enter password
        pass_box = self.sleepy_find_element(By.ID, 'password')
        pass_box.send_keys(password)
        logging.info('Filled password box')
        # Click continue
        pass_box.send_keys(Keys.ENTER)
        time.sleep(1)
        logging.info('Logged in')
    
        # NEW 
        # Click Okey Let's go for ChatGPT: Tips for getting started - Dialog
        OkeyLetsGo_button = self.sleepy_find_element(By.XPATH, '//div[text()="Okay, let’s go"]')
        OkeyLetsGo_button.click()
        time.sleep(1)
        logging.info('Clicked Okay, let’s go button')
        # END NEW
    
        try:
            # Pass introduction

```
switch_model works now too (little bit hard coded, but tested):

```
from selenium.webdriver.common.action_chains import ActionChains
...
def switch_model(self, model_name : str, mode : str = 'Default'):
        '''
        Switch the model for ChatGPT+ users.

        Args:
            model_name: str = The name of the model, either GPT-3.5 or GPT-4
            mode: str = For GPT-4 : Name of the Mode: either 'Default', 'Code Interpreter' or 'Plugins'
        Returns:
            bool: True on success, False on fail
        '''
        if model_name in ['GPT-3.5', 'GPT-4']:
            logging.info(f'Switching model to {model_name}')
            try:
                if model_name=='GPT-3.5':
                    self.browser.find_element(By.XPATH, self.gpt_xq.format(model_name)).click()
                    return True
                elif model_name=='GPT-4':
                    element = self.browser.find_element(By.XPATH, self.gpt_xq.format(model_name))
                    actions = ActionChains(self.browser)
                    # only move mouse for mouse over
                    actions.move_to_element(element).perform()
                    # now select the right mode
                    modeElement = self.browser.find_element(By.XPATH, '//span[text()="{}"]'.format(mode))
                    modeElement.click()
                    return True
                else:
                    return False
            except Exceptions.NoSuchElementException:
                logging.error('Button is not present')
        return False

```
would be great if you could take the code adapted

**Comment by ugorsahin** [comment]
It is interesting that I can pass the info screen on my end but not you, I will take a look at this later on.

**Comment by ugorsahin** [comment]
This problem is now fixed with 30606eabc69b3d9dbe0f8dfef94687b84032525d, the id selector was not working properly, now we skip the tutorial page without clicking anything. Thank you!
